processor/test/operation.service.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { describe, test, expect, afterEach } from '@jest/globals';
+import {
+  ConfigResponse,
+  OperationService,
+  OperationServiceOptions,
+  ModifyPayment,
+} from '../src/services/types/operation.type';
+import { DefaultOperationService } from '../src/services/operation.service';
+import { MockOperationProcessor } from '../src/services/processors/mock-operation.processor';
+import { paymentSDK } from '../src/payment-sdk';
+import { DefaultPaymentService } from '@commercetools/connect-payments-sdk/dist/commercetools/services/ct-payment.service';
+import { mockGetPaymentResult, mockUpdatePaymentResult } from './utils/mock-payment-data';
+import sinon from 'sinon';
```

Comment:
just curiosity: doesn't jest provides same [features](https://jestjs.io/docs/mock-function-api)? what are the benefits of adding `sinon`?

Reply:
just get used to use `sinon` for mocking purpose and not familiar with `jest`.
If `jest` provides same feature, i agree we would better avoid duplicated dependencies for same purpose.
https://github.com/commercetools/connect-payment-integration-template/pull/52/commits/39c5294ff6a54468336ea38dae70b679bfb81599